connectors/citrus-kubernetes/src/main/resources/org/citrusframework/schema/citrus-kubernetes-config.xsd

```diff
@@ -30,6 +30,7 @@
       <xs:attribute name="version" type="xs:string"/>
       <xs:attribute name="username" type="xs:string"/>
       <xs:attribute name="password" type="xs:string"/>
+      <xs:attribute name="oauthToken" type="xs:string"/>
```

Comment:
the `KubernetesClientParser` misses this property for copying.

Reply:
The parser contains already the property